### ISSUE TITLE
How can I set up L2 weight decay and activation functions for a a fully connected NN? 

### ISSUE THREAD

**Issue by chf42** [deepchem/deepchem#1680]
**Description:**

- DeepChem Version: 2.1.1
- Platform: CentOS Linux 7.6

In DeepChem, how can I set up L2 weight decay and activation functions for a a fully connected NN?

Below is my code with 3-hidden layers TensorGraphMultiTaskRegressor. I set up "weight_decay_penalty = .0004", but I am not sure whether it is the correct way to set the weight_decay as 0.0004. Besides, I used "activation_fns = tf.nn.relu" to set up ReLU activation functions for each layer. I wonder if there is a way to set up different activation functions for different layers? 

Also, is there any other arguments I can set up for  dc.models.TensorGraphMultiTaskRegressor?

Thanks,
Cheng

```
model = dc.models.TensorGraphMultiTaskRegressor(
	len(tasks),
	n_features,
	layer_sizes=[512,256,256],
	weight_init_stddevs=[.02,.02,.02],
	bias_init_consts = [1.,1.,1.],
	**weight_decay_penalty = .0004,**
	weight_decay_penalty_type="l2",
	dropouts=[.25,.25,.25],
        **activation_fns=tf.nn.relu,**
	learning_rate=0.001,
	batch_size=128,
	seed=5758)
```

**Comment by chf42** [comment]
@peastman Thank you so much! It really help. One last question before I close this issue. 

Is there a way to set batch normalization for MultitaskRegressor? I searched the help documents, and didn't find the arguments.

Thanks.
Cheng

**Comment by rbharath** [comment]
@chf42 I believe the canned `MultitaskRegressor` doesn't support this, but we've completed our conversion to using Keras models. So you can just add batch normalization to a multitask deep network that you implement using Keras layers
